index.html

```diff
@@ -11,22 +11,28 @@
 </head>
 
 <body>
+   
+
     <div class="wrapper">
-        <nav class="nav-bar-desktop">
-            <a class="logo" href="#">Hamma Azarok</a>
-            <ul class="nav-list">
-                <li class="nav-item"><a href="#portfolio">Portofolio</a></li>
-                <li class="nav-item"><a href="#about">About</a></li>
-                <li class="nav-item"><a href="#contact">Contact</a></li>
+    <!-- Create the Navigation menu this way -->
+    <header class="nav-header ">
+        <a href="#" class="nav-log">
+            <img src="img/logo.png" />
+        </a>
+        <nav id="navbar">
+            <ul class="nav-menu blur">
+                <li class="nav-item"><a href="#" class="nav-link">Home</a></li>
+                <li class="nav-item"><a href="#portfolio" class="nav-link">Portofolio</a></li>
+                <li class="nav-item"><a href="#about" class="nav-link">About</a></li>
+                <li class="nav-item"><a href="#contact" class="nav-link">Contact</a></li>
             </ul>
-        </nav>
-        <header class="nav-bar">
-            <a href="#">
-                <p>Hamma Azarok</p>
+            <a href="#" class="hamburger">
+                <span class="src-only" hidden="true">Open  main menu</span>
+                <i class="fas fa-bars"></i>
```

Comment:
- [ ] Good job by adding the blur in the background of the mobile menu. But as the previous reviewer mentioned in the previous comment to stick the background of the mobile menu to the one from Fimga, I think you missed one detail on the height. I just noticed that from the Figma template the background of the mobile menu covers the entire page i.e. from header to footer. I think once you fixed this issue your milestone will be completed. I know you can do it.